internal/build/gotool.go

```diff
@@ -55,7 +55,7 @@ func (g *GoToolchain) Go(command string, args ...string) *exec.Cmd {
 	}
 	// CKZG by default is not portable, append the necessary build flags to make
 	// it not rely on modern CPU instructions and enable linking against
-	tool.Env = append(tool.Env, "CGO_CFLAGS=-D__BLST_PORTABLE__")
+	tool.Env = append(tool.Env, "CGO_CFLAGS=-std=c11 -O2 -D__BLST_PORTABLE__")
```

Comment:
You need to surround the values in quotation marks. Otherwise it's treated as multiple flags.
```suggestion
	tool.Env = append(tool.Env, "CGO_CFLAGS=\"-std=c11 -O2 -D__BLST_PORTABLE__\"")
```

Reply:
I don't think it's required here. The process environment is a slice of strings and everything after the first `=` is the variable value. The env slice is not interpreted by shell.